tests/SprykerSdkTest/Zed/AppSdk/Communication/Console/ValidateAsyncApiConsoleTest.php

```diff
@@ -0,0 +1,62 @@
+<?php
+
+/**
+ * Copyright © 2019-present Spryker Systems GmbH. All rights reserved.
+ * Use of this software requires acceptance of the Evaluation License Agreement. See LICENSE file.
+ */
+
+namespace SprykerSdkTest\Zed\AppSdk\Communication\Console;
+
+use Codeception\Test\Unit;
+use SprykerSdk\Zed\AppSdk\Communication\Console\AbstractConsole;
+use SprykerSdk\Zed\AppSdk\Communication\Console\ValidateAsyncApiConsole;
+use Symfony\Component\Console\Output\OutputInterface;
+
+/**
+ * @group SprykerSdkTest
+ * @group Zed
+ * @group AppSdk
+ * @group Communication
+ * @group Console
+ * @group ValidateAsyncApiConsoleTest
+ */
+class ValidateAsyncApiConsoleTest extends Unit
+{
+    /**
+     * @var \SprykerSdkTest\Zed\AppSdk\CommunicationTester
+     */
+    protected $tester;
+
+    /**
+     * @return void
+     */
+    public function testHaveValidAsyncApiFile(): void
+    {
+        // Arrange
+        $this->tester->haveInvalidAsyncApiFileWithDuplicatedMessageNames();
```

Comment:
PLease check all your tests as the method name states a test for a valid file but the test code creates an invalid file...